### PR TITLE
7 add topas simulations

### PR DESCRIPTION
**Pull request by Villadslj**
Hej

Here is the Topas simultions of the 160 MeV, 160 MeV 1 MU, 2 Gy SOBP and 100 Gy SOBP.

There is plots for 2 Gy SOBP.

-Villads

### CODE REVIEW

File: TOPAS/Plots/2GySOBP/z-plot.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from topas2numpy import BinnedResult
+
+files = []
+files.append(BinnedResult('/home/villads/2022_DCPT_LET/TOPAS/output/protons.592479/Scoring_protonD_ZBox.csv'))
```
Comment (grzanka) [inline TOPAS/Plots/2GySOBP/z-plot.py:6]: please avoid storing paths to local folders, the code should work without any modifications by any user

File: TOPAS/Plots/2GySOBP/z-plot.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+files.append(BinnedResult('/home/villads/2022_DCPT_LET/TOPAS/output/protons.592479/Scoring_protonLET_ZBox.csv'))
+for data in files:
+    print('{0} [{1}]'.format(data.quantity, data.unit))
+    print('Statistics: {0}'.format(data.statistics))
```
Comment (grzanka) [inline TOPAS/Plots/2GySOBP/z-plot.py:10]:
```suggestion
    print(f'Statistics: {data.statistics}')
```

better to use f-strings

**Comment by grzanka** [comment]
@Villadslj please adhere to the directory structure mentioned here: https://github.com/APTG/2022_DCPT_LET#directory-structure

**Comment by grzanka** [comment]
Couple of remarks:
- [ ] Merge the latest changes from main into this branch. Now this branch is marked as out-of-date
- [ ] Remove DICOM files from the TOPAS input directory. Use the files directly from `data/resources` We do not wan't to have these files to be stored twice in the repository, especially pretty large `RD*dcm`. The aim of this repository is that the users (physicists) would clone the full repository (will have access to all files, in all directories) and be able to rerun simulations with their own TOPAS installation.
- [ ] Move `*png` files from `data/reports/figures/*/*.png` to `data/topas/results`. In the `reports/figures` we plan to add a comparison between various simulations (note that the main README mentions `reports/figures`, not `data/reports/figures`
- [ ] Can you replace hardcoded paths in `slurm` like to your home directories to $HOME ?
- [ ] Remove `--mail-user=villads.lundsteen@gmail.com` from SLURM, otherwise you will be spammed by submissions from other users
- [ ] Rename `slap` to `slab` in TOPAS configs
- [ ] `data/resources` should contain data not related to any particular MC code.  Move `topas_beam` to `data/topas/input`
- [ ] Remove hardcoded paths like `/TopasDCPT/material_assignment/SPRtoMaterial__Brain.txt` from TOPAS input beam files
- [ ] Include in the repo a script which reads DICOM files and generates TOPAS beam files

**Comment by grzanka** [comment]
@Villadslj what is the coordinate system you are using: IEC with beam going towards positive/negative Y axis ? or is it "MC convention" system when beams goes along positive Z ? This is crucial for interpreting the results.

**Comment by Villadslj** [comment]
> 
We have an in-house software program to convert the dicom files to TOPAS files, (Not sure if I can share it, will check).
As for the having the beam in the negative Y-axis, the dicom plan has the gantry in in 0 degree position, so that is why.
did not want to rotate the the whole beam, bench and everything

**Comment by grzanka** [comment]
> > 
> 
> We have an in-house software program to convert the dicom files to TOPAS files, (Not sure if I can share it, will check). As for the having the beam in the negative Y-axis, the dicom plan has the gantry in in 0 degree position, so that is why. did not want to rotate the the whole beam, bench and everything

I think it's important point. With SHIELD-HIT12A I simulate the beam going along Z axis, as this is most commonly used convention in MC codes. I suspect other codes (i.e. Fluka) may do  the same.
Having beam along Z axis would then simplify comparison between various MC codes.

For comparison with TPS I would re-arange the data during binary data -> DICOM transition.

@nbassler what is your opinion ?

**Comment by Villadslj** [comment]
> Have you commited all the changes marked as done here:
> 
> image
> 
> I still see the old version here:
> 
> image

I have now... (sorry)

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**